packages/react-keytips/src/utilities/isDisabled.ts

```diff
@@ -0,0 +1,7 @@
+export const isDisabled = (target?: HTMLElement): boolean => {
+  if (!target) return false;
+
+  return (
+    target?.hasAttribute('disabled') || target.hasAttribute('aria-disabled')
```

Comment:
```suggestion
    target.hasAttribute('disabled') || target.hasAttribute('aria-disabled')
```

Reply:
Should we also support the `aria-disabled="false"` case?

Reply:
@dmytrokirpa I also thought about this, but currently usually it's not set explicitly to false, if it's not disabled, the aria or disabled attribute are not present. To make it more bulletproof, we can check if it's true.